sql/core/pom.xml

```diff
@@ -252,6 +252,11 @@
       <artifactId>htmlunit3-driver</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
should add this to dependency management first

Reply:
For non-ASF ones, we will include their licenses in our licenses-binary directory and LICENSE-binary file, and also update the NOTICE-binary document for any applicable notices.